generic/rcutorture.py

```diff
@@ -39,16 +40,13 @@ def setUp(self):
         """
         self.results = []
         self.log.info("Check if CONFIG_RCU_TORTURE_TEST is enabled\n")
-        ret = os.system(
-            'cat /boot/config* | grep CONFIG_RCU_TORTURE_TEST=m')
-        if ret != 0:
+        ret = linux_modules.check_kernel_config('CONFIG_RCU_TORTURE_TEST')
+        if ret == 0:
```

Comment:
the module can return 0/1/2, but what I need is '0', so it should be ;
if linux_modules.check_kernel_config('CONFIG_RCU_TORTURE_TEST') == 0: